src/utils/temp_folder.test.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+import { cleanUpTempFolder, getTempFolder } from './temp_folder';
+import * as fs from 'fs';
+
+describe('temp folder test', () => {
+	it('temp returns a folder that exists', () => {
+		const tempFolderPath = getTempFolder();
+		expect(fs.existsSync(tempFolderPath)).to.equal(true);
+	});
+
+	it('temp path returned contains the correct subfolders', () => {
+		const tempFolderPath = getTempFolder();
+		expect(tempFolderPath).to.contains('ardrive');
+		expect(tempFolderPath).to.contains('temp');
+	});
+
+	it('cleanUpTempFolder removes the temporary folder from the local system', () => {
+		const tempFolderPath = getTempFolder();
+		expect(fs.existsSync(tempFolderPath)).to.equal(true);
+		cleanUpTempFolder();
+		expect(fs.existsSync(tempFolderPath)).to.equal(false);
+	});
+});
```

Comment:
Please add 1 or more additional tests that verify that cleanupTempFolder can either (or both) be called twice without error or be called when there's nothing to clean up without error.